sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubBufferedProducerAsyncClient.java

```diff
@@ -81,7 +92,62 @@
+ * events can fit into a batch.</p>
+ *
+ * <p>NOTE that {@code Mono<Integer>} returned must be subscribed to, or eventually subscribed to if chained to
+ * reactive operators in order to start the operation.</p>
+ *
+ * <!-- src_embed com.azure.messaging.eventhubs.eventhubbufferedproducerasyncclient.enqueueEvents-iterable -->
+ * <pre>
+ * TokenCredential credential = new DefaultAzureCredentialBuilder&#40;&#41;.build&#40;&#41;;
+ *
+ * &#47;&#47; &quot;&lt;&lt;fully-qualified-namespace&gt;&gt;&quot; will look similar to &quot;&#123;your-namespace&#125;.servicebus.windows.net&quot;
+ * &#47;&#47; &quot;&lt;&lt;event-hub-name&gt;&gt;&quot; will be the name of the Event Hub instance you created inside the Event Hubs namespace.
+ * EventHubBufferedProducerAsyncClient client = new EventHubBufferedProducerClientBuilder&#40;&#41;
+ *     .credential&#40;&quot;fully-qualified-namespace&quot;, &quot;event-hub-name&quot;, credential&#41;
+ *     .onSendBatchSucceeded&#40;succeededContext -&gt; &#123;
+ *         System.out.println&#40;&quot;Successfully published events to: &quot; + succeededContext.getPartitionId&#40;&#41;&#41;;
+ *     &#125;&#41;
+ *     .onSendBatchFailed&#40;failedContext -&gt; &#123;
+ *         System.out.printf&#40;&quot;Failed to published events to %s. Error: %s%n&quot;,
+ *             failedContext.getPartitionId&#40;&#41;, failedContext.getThrowable&#40;&#41;&#41;;
+ *     &#125;&#41;
+ *     .buildAsyncClient&#40;&#41;;
+ *
+ * List&lt;EventData&gt; events = Arrays.asList&#40;new EventData&#40;&quot;maple&quot;&#41;, new EventData&#40;&quot;aspen&quot;&#41;,
+ *     new EventData&#40;&quot;oak&quot;&#41;&#41;;
+ *
+ * &#47;&#47; Enqueues the events to be published.
+ * client.enqueueEvents&#40;events&#41;.subscribe&#40;numberOfEvents -&gt; &#123;
+ *     System.out.printf&#40;&quot;There are currently: %d events in buffer.%n&quot;, numberOfEvents&#41;;
+ * &#125;, error -&gt; &#123;
+ *         System.err.println&#40;&quot;Error occurred enqueueing events: &quot; + error&#41;;
+ *     &#125;,
+ *     &#40;&#41; -&gt; &#123;
+ *         System.out.println&#40;&quot;Events successfully enqueued.&quot;&#41;;
+ *     &#125;&#41;;
+ *
+ * &#47;&#47; Seconds later, enqueue another event.
+ * client.enqueueEvent&#40;new EventData&#40;&quot;bonsai&quot;&#41;&#41;.subscribe&#40;numberOfEvents -&gt; &#123;
+ *     System.out.printf&#40;&quot;There are %d events in the buffer.%n&quot;, numberOfEvents&#41;;
+ * &#125;, error -&gt; &#123;
+ *         System.err.println&#40;&quot;Error occurred enqueueing events: &quot; + error&#41;;
+ *     &#125;,
+ *     &#40;&#41; -&gt; &#123;
+ *         System.out.println&#40;&quot;Event successfully enqueued.&quot;&#41;;
+ *     &#125;&#41;;
+ *
+ * &#47;&#47; Causes any buffered events to be flushed before closing underlying connection.
+ * client.close&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.messaging.eventhubs.eventhubbufferedproducerclient.enqueueEvents-iterable -->
```

Comment:
Add see tags at end to:
1. Builder
2. the EH package com.azure.messaging.eventhubs